easybuild/easyconfigs/c/COSMO_pompa/COSMO_pompa-f9916ba-28.11.19-CrayCCE-19.08-cordex-double.eb

```diff
@@ -0,0 +1,49 @@
+# kraushm, omlins (CSCS)
+easyblock = 'MakeCp'
+
+name = 'COSMO_pompa'
+version = 'f9916ba'
+versionsuffix = '-28.11.19-cordex-double'
```

Comment:
Do we mind, that the version suffix of the module is not equal the file name?

Reply:
Yes, I agree that it is not optimal, right now. As discussed offline, I think it logically rather belongs to the version (it is the essentially the same information as the hash) as Guilherme originally suggested. In order to avoid changing all the filenames that depend on the version, we can split create local variables for the hash and for the date and use them for the filenames... I suggest to apply these changes however in the scope of a general cleanup when creating the PE 19.11 recipes.

Reply:
Doing this "cleanup" with PE19.11 is fine with me.